docs/install.rst

```diff
@@ -29,12 +29,11 @@ version yourself with:
 
 Then, just use whichever of those Pythons claim to be 3.6 or higher.
 
-Option 1: Install from PyPi with `pip`
---------------------------------------
-The latest release of RepoBee is on PyPi, and can thus be installed as usual with pip.
-I strongly discourage system-wide pip installs (e.g. ``sudo pip install <package>``), as this
-may land you with incompatible packages in a very short amount of time. A per-user install
-can be done like this:
+Option 1: Install from PyPi with `pip` --------------------------------------
+The latest release of RepoBee is on PyPi, and can thus be installed as usual
+with pip. I strongly discourage system-wide pip installs (e.g. ``sudo pip
+install <package>``), as this may land you with incompatible packages in a very
+short amount of time. A per-user install can be done like this:
```

Comment:
The formatting here has gone a bit bonkers. I'm guessing it's your plugin to limit line count. We could try adding a blank line between the heading and the content to avoid that happening.
```suggestion
Option 1: Install from PyPi with `pip` 
--------------------------------------

The latest release of RepoBee is on PyPi, and can thus be installed as usual
with pip. I strongly discourage system-wide pip installs (e.g. ``sudo pip
install <package>``), as this may land you with incompatible packages in a very
short amount of time. A per-user install can be done like this:
```